pkg/fusortest/util.go

```diff
@@ -94,6 +94,23 @@ func AssertNotNil(t *testing.T, a interface{}, message ...string) {
 	t.Fatal(msg)
 }
 
+// AssertNil - Assert input is nil
+func AssertNil(t *testing.T, a interface{}, message ...string) {
```

Comment:
Thanks I was lazy and kept using `ft.AssertTrue(t, obj == nil, "msg")`. +1 to `AssertNil`